ui/app.js

```diff
@@ -4510,6 +4510,31 @@ $(document).ready(() => {
                 </div>`
         );
       });
+      $(".contextmenu").on("click", ".incidents-delete", function () {
+        $(".incidents-items")
+          .find("[data-id='" + $(this).data("info") + "']")

```

Comment:
nit: using back ticks can make this a bit simpler
```suggestion
          .find(`[data-id="${$(this).data("info")}"]`)

```

Reply:
done <3